src/parser/druid/restoration/modules/features/Lifebloom.js

```diff
@@ -16,7 +16,10 @@ import { t } from '@lingui/macro';
 
 class Lifebloom extends Analyzer {
   get uptime() {
-    return this.combatants.getBuffUptime(SPELLS.LIFEBLOOM_HOT_HEAL.id);
+    // Only either LIFEBLOOM_HOT_HEAL or LIFEBLOOM_DTL_HOT_HEAL can be up (with or without the DTL legendary), but
+    // DTL Lifeblooms (LIFEBLOOM_DTL_HOT_HEAL) are on two targets so their BuffUptime need to behalved for a percentage
+    return this.combatants.getBuffUptime(SPELLS.LIFEBLOOM_HOT_HEAL.id)
+      + (this.combatants.getBuffUptime(SPELLS.LIFEBLOOM_DTL_HOT_HEAL.id) / 2);
```

Comment:
You don't need to divide by two here. combatants.getBuffUptime gives you what you want even if two are out at the same time.

Reply:
checked the wrong getBuffUptime function... fixed